_config.yml

```diff
@@ -273,28 +273,29 @@ enableHackathon: true
 # Hackathon About Block
 aboutHackathonTitle: "CollaborationFest"
 hackathonRows:
- - {title: "What is it?", image: "hackathon-0.png", details: "BCC2020 will finish with CollaborationFest, a 2 (or 4) day collaborative work event where our community gathers to contribute code, documentation, training materials, and challenging analysis problems and use cases. If you are interested in learning and contributing in an intensely collaborative environment, then CollaborationFest is for you."}
+ - {title: "What is it?", image: "hackathon-0.png", details: "BCC2020 will finish with an online CollaborationFest, a 2 (or 4) day collaborative work event where our community gathers virtually to contribute code, documentation, training materials, and challenging analysis problems and use cases. If you are interested in learning and contributing in an intensely collaborative environment, then CollaborationFest is for you."}
  - {title: "Core and Encore", image: "hackathon-0.png", details: "CoFest is split in half again this year:<br/><b>CollaborationFest: Core: Friday & Saturday</b>: There will be lots of collaborative work, but also lots of learning about projects and software. This will be the larger of the two stages. All you need to attend this is an interest and willingness to contribute.<br/><b>CollaborationFest: Encore: Sunday & Monday</b>: The smaller (and dare we say <i>more intense</i>) of the two stages, these two days will focus on collaboration, collaboration, collaboration."}
- - {title: "Registration", image: "hackathon-0.png", details: "CollaborationFest is free, for all 4 days. You still need to register (and select either 2 or 4 days). You will need to cover some, and possibly all, of your meal expenses. Organizers will provide coffee / beverages throughout. We will attempt to get funding for as many lunches as possible."}
- - {title: "Preparing for CoFest", image: "hackathon-0.png", details: "Add or expand the cluster ideas document (coming)<br><br>Add an introduction and indicate your preferred working clusters (coming)<br><br>Ask questions on the Conference gitter channel"}
+ - {title: "Registration", image: "hackathon-0.png", details: "CollaborationFest is free, for all 4 days. You still need to register (and select either 2 or 4 days)."}
+ - {title: "Preparing for CoFest", image: "hackathon-0.png", details: "<a href='https://docs.google.com/document/d/1UctntMyOMOFlyxgvZPiWqgXz7R3FjPKnuBVMoNxl9Jc/edit?usp=sharing'>Contribute to the cluster ideas document defining community goals for the two days</a><br><br><a href='https://docs.google.com/spreadsheets/d/1AoSztstDDiJphefjnp7hvZByZKavyusIHzQO3-qAC7M/edit?usp=sharing'>Introduce yourself and indicate your preferred working clusters</a><br><br>Ask questions on the <a href='https://bcc.zulipchat.com/'>Bioinformatics Community Conference Zulip Chat</a>"}
```

Comment:
two days &rarr; four days

Reply:
Good catch, thanks so much. Tried to generalize the language.